float64[] x
float64[] y
string[] Target_Type
int16[] IDed
int16[] Served
int64[] Points
